apollo-router/src/query_planner/snapshots/apollo_router__query_planner__tests__typename_propagation2.snap.new

```diff
@@ -0,0 +1,28 @@
+---
+source: apollo-router/src/query_planner/tests.rs
+assertion_line: 1528
+expression: "serde_json::to_value(&response).unwrap()"
+---
+{
+  "data": {
+    "book": null
+  },
+  "errors": [
+    {
+      "message": "couldn't find mock for query {\"query\":\"query QueryBook__node_relay_subgraph__2($representations:[_Any!]!){_entities(representations:$representations){...on Book{__typename id}}}\",\"operationName\":\"QueryBook__node_relay_subgraph__2\",\"variables\":{\"representations\":[{\"__typename\":\"Book\",\"bookId\":\"book1\",\"author\":null}]}}",
```

Comment:
Hmm that looks suspicious... Maybe the query plan changed recently with a federation upgrade ?